newIDE/app/src/UI/Menu/ContextMenu.js

```diff
@@ -95,6 +99,9 @@ const MaterialUIContextMenu = React.forwardRef<
         left: anchorPosition[0],
         top: anchorPosition[1],
       }}
+      style={{
+        zIndex: itemAboveBlockingLayerZIndex,
```

Comment:
Menus are now always above the blocking layer, so they can be interacted with during inapptutorials

Reply:
There is no display issue when filling out expression parameters after selecting the expression from the context menu ? (dialog over context menu)

Reply:
No because the context menu is closed automatically when the dialog opens